Guided Hunting - Azure Resource Explorer.ipynb

```diff
@@ -0,0 +1,7311 @@
+{
```

Comment:
<p>I would just merge this with the previous cell (add before the final print("Graph notes...)) - since it doesn't really do anything that the user needs to see output for.</p><p>(or add it to beginning of next cell)</p><p>Separate it from the prev code with space and comments. The #Create Graph comment isn't accurate though is it?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Azure/Azure-Sentinel-Notebooks/pull/127/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='40'/>

Reply:
<p>added to following cell!</p>